### ISSUE TITLE
Document how to use with Cocoapods

### ISSUE THREAD

**Issue by tmandry** [tmandry/Swindler#34]
0.1.0 will be the first version actually published on Cocoapods. Need to add something to the README about adding it to your Podfile.

**Comment by natew** [comment]
In the spirit of this, I don't see 0.0.3 on cocoapods after an update, is it published?

**Comment by tmandry** [comment]
Yeah I've been putting off publishing because of all the breaking changes I had planned prior to 0.1.0. The last I had planned was #17 upgrade to PromiseKit 6, which is in master. So 0.1.0 should be out soon.
